premis_event_service/templates/premis_event_service/agent.html

```diff
@@ -1,89 +1,89 @@
+        {% for agent in agents %}
+            <tr>
+                <td>
+                    <i class="icon-tag"></i> <a href='http://{{ request.META.HTTP_HOST }}/agent/{{ agent.agent_identifier }}'>http://{{ request.META.HTTP_HOST }}/agent/{{ agent.agent_identifier }}</a>
+                </td>
+                <td>
+                    {{ agent.agent_name }}
+                </td>
+                <td>
+                    {% if agent.agent_type == 'Software' %}
+                    <i class="icon-save"></i>
+                    {% else %}{% if agent.agent_type == 'Organization' %}
+                    <i class="icon-leaf"></i>
+                    {% else %}{% if agent.agent_type == 'Event' %}
+                    <i class="icon-calendar"></i>
+                    {% else %}{% if agent.agent_type == 'Personal' %}
+                    <i class="icon-user"></i>
+                    {% endif %}{% endif %}{% endif %}{% endif %}
+                    {{ agent.agent_type }}
+                </td>
+            </tr>
+        {% endfor %}
+    </table>
+{% elif agents|length == 1 %}
+    <table class="table table-striped info">
+        <colgroup>
+            <col class="labels">
+            <col class="data">
+        </colgroup>
+        <h3 class="main"><i class="icon-white icon-flag"></i> http://{{ request.get_host }}{% url 'premis_event_service.views.humanAgent' agents.0.agent_identifier %}</h3>
+            <tr>
+                <th style="width: 20%;">Agent ID:</th>
+                <td><i class="icon-tag"></i> <a href="http://{{ request.get_host }}{% url 'premis_event_service.views.humanAgent' agents.0.agent_identifier %}">http://{{ request.get_host }}{% url 'premis_event_service.views.humanAgent' agents.0.agent_identifier %}</a></td>
+            </tr>
+            <tr>
+                <th style="width: 20%;">Agent Type:</th>
+                <td>
+                    {% if agents.0.agent_type == 'Software' %}
+                    <i class="icon-save"></i>
+                    {% else %}{% if agents.0.agent_type == 'Organization' %}
+                    <i class="icon-leaf"></i>
+                    {% else %}{% if agents.0.agent_type == 'Event' %}
+                    <i class="icon-calendar"></i>
+                    {% else %}{% if agents.0.agent_type == 'Personal' %}
+                    <i class="icon-user"></i>
+                    {% endif %}{% endif %}{% endif %}{% endif %}
+                    {{ agents.0.agent_type }}
+                </td>
+            </tr>
+            <tr>
+                <th style="width: 20%;">Agent Name:</th>
+                <td>{{ agents.0.agent_name }}</td>
+            </tr>
+            <tr>
+                <th style="width: 20%;">Agent Note:</th>
+                <td>{{ agents.0.agent_note }}</td>
+            </tr>
+    </table>
+    <a class="btn btn-success btn-large" href="http://{{ request.META.HTTP_HOST }}/APP/agent/{{ agents.0.agent_identifier }}/">
```

Comment:
Perhaps we should either use reverse here or use some logic to determine the scheme being used? Otherwise, this will be incorrect if we switch the site to https, or change the path for some reason.